### PR TITLE
Removed redundant fde kernel installation code

### PR DESCRIPTION
**Pull request by SRIKKANTH**
Moving .efi files is no longer needed.

### CODE REVIEW

File: lisa/transformers/kernel_installer.py [hunk 1]
```diff
@@ -140,37 +142,17 @@ def _internal_run(self) -> Dict[str, Any]:
             if installer.runbook.source != "linux-image-azure-fde":
                 posix = cast(Posix, node.os)
                 posix.replace_boot_kernel(installed_kernel_version)
-            else:
```
Comment (squirrelsc) [inline lisa/transformers/kernel_installer.py:143]: Please check with the code author understand if it's redundant. Is it related to cvm?

Comment (SRIKKANTH) [reply lisa/transformers/kernel_installer.py:143]: @LiliDeng This CVM logic is no longer required and causing failures as it cannot find .efi files under "/usr/lib/linux/efi/".

Comment (LiliDeng) [reply lisa/transformers/kernel_installer.py:143]: will you run old distro which needs this logic?

Comment (SRIKKANTH) [reply lisa/transformers/kernel_installer.py:143]: No, we only use this with proposed kernel kernels. This logic is not needed on all recent CVM kernels on both focal and jammy.

File: lisa/transformers/kernel_installer.py [hunk 2]
```diff
@@ -140,37 +142,17 @@ def _internal_run(self) -> Dict[str, Any]:
...
         node.reboot()
+        kernel_version_after_install = uname.get_linux_information(force_run=True)
         self._log.info(
             f"kernel version after install: "
-            f"{uname.get_linux_information(force_run=True)}"
+            f"{kernel_version_after_install}",
         )
+        assert_that(
+            kernel_version_after_install, "Kernel installation Failed"
+        ).is_equal_to(kernel_version_before_install)
```
Comment (LiliDeng) [inline lisa/transformers/kernel_installer.py:152]: @squirrelsc long time ago, we had a discussion on this check, do you think we can add the check?

File: lisa/transformers/kernel_installer.py [hunk 3]
```diff
@@ -140,37 +142,14 @@ def _internal_run(self) -> Dict[str, Any]:
             if installer.runbook.source != "linux-image-azure-fde":
```
Comment (LiliDeng) [inline lisa/transformers/kernel_installer.py:142]: we should remove if

Comment (SRIKKANTH) [reply lisa/transformers/kernel_installer.py:142]: That step is not needed for cvm. So lets keep it.

File: lisa/transformers/kernel_installer.py (hunk 2 above)
Comment (squirrelsc) [reply lisa/transformers/kernel_installer.py:152]: Yes, it looks good to me.

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**